Core/src/main/java/com/plotsquared/core/PlotSquared.java

```diff
@@ -645,7 +645,7 @@ private void sortPlotsByHash(final @NonNull Plot @NonNull [] input) {
         } else {
             list = new ArrayList<>(input);
         }
-        list.sort(Comparator.comparingLong(a -> ExpireManager.IMP.getTimestamp(a.getOwnerAbs())));
+        list.sort(Comparator.comparingLong(a -> PlotSquared.platform().expireManager().getTimestamp(a.getOwnerAbs())));
```

Comment:
We've had performance issues with guice before (https://github.com/IntellectualSites/PlotSquared/pull/3315), so I think frequent accesses should be avoided, e.g.
```suggestion
        ExpireManager expireManager = PlotSquared.platform().expireManager();
        list.sort(Comparator.comparingLong(a -> expireManager.getTimestamp(a.getOwnerAbs())));
```